ipinfo/main.go

```diff
@@ -69,6 +69,8 @@ func main() {
 		err = cmdMmdb()
 	case cmd == "download":
 		err = cmdDownload()
+	case cmd == "tool":
```

Comment:
Remember to add a help message to the root command for this. (cmd_default IIRC).